release/ray_release/tests/test_retry.py

```diff
@@ -0,0 +1,57 @@
+from ray_release import retry
```

Comment:
add this test in `release/BUILD.bazel` file?

Reply:
Nice! Thanks! Added. I miss the hermetic build.